bundles/org.openhab.ui/web/src/pages/settings/pages/home/home-edit.vue

```diff
@@ -83,7 +82,9 @@
                 </f7-list-item>
               </f7-list>
             </div>
+          </f7-block>
 
+          <f7-block style="z-index: -1">
```

Comment:
FYI I had to switch from col to block to make z-index work, and I had to wrap the bottom settings in their own block, which IMO makes sense anyway.